### DISCUSSION TITLE
How to determine scaling

### DISCUSSION THREAD

**Discussion by PEI0214** [idaholab/moose#27102] [Q&A Modules: Porous Flow] [answered]
Hi，
I saw the Convergence criteria in the porous flow module, but I still don't understand how to determine scaling. Can someone give an example of determining scaling?

Thank you for your help.

**Comment by GiudGiud** [comment]
Hello

You determine scaling usually by looking at the equation residuals. See the option in the Debug block
Is automatic_scaling , a parameter to set in the executioner, not working for you?

Guillaume

**Reply by PEI0214** [reply]
When I use automatic_scaling, the computed results become very small. What could be the reason for this?

**Reply by GiudGiud** [reply, marked as answer]
hello

scaling should not affect the results. It should affect the size of the residuals, especially when comparing between variables. 
Which variables do you have?
Without scaling, what does this show?
```
[Debug]
  show_var_residual_norms = true
```
With automatic scaling, what does it show?

Did you read this page?
https://mooseframework.inl.gov/moose/source/systems/NonlinearSystemBase.html#scaling

Guillaume

**Reply by PEI0214** [reply]
This problem has been solved, thank you very much for your answer.
